flasgger_marshmallow/__init__.py

```diff
@@ -238,9 +238,8 @@ def wrapper(*args, **kw):
                     if r_headers_schema:
                         headers = r_headers_schema().load(headers or {})
             except Exception as e:
```

Comment:
@bogdancondurache I'm considering removing this try/except because if we're not serializing correctly, we shouldn't hide it, but will return 500 instead... wdyt?

Reply:
I think this part is ok, considering it will return an error to the client with a 400 status code, which seems more fit than 500. But considering we also need telemetry, maybe it makes sense to remove try/excepts from here in order to catch them in our code to also log the exceptions

Reply:
talked offline... removed the try/except and we will get alerts for unhandled exceptions (which should have been caught by tests)